src/containers/WebComponentLoader.test.js

```diff
@@ -315,6 +315,7 @@ describe("When no user is in state", () => {
               instructions={instructions}
               authKey={authKey}
               theme="light"
+              useCache={true}
```

Comment:
❓ I don't think this is doing anything. The prop is `loadCache` and I _think_ the test below are just picking up the default